packages/core/src/internal/io.js

```diff
@@ -52,7 +52,7 @@ export function put(channel, action) {
   }
   if (is.undef(action)) {
     action = channel
-    channel = null
+    channel = undefined
```

Comment:
a short comment above this line that we assign undefined and not null to make default parameter work at later point would be helpful